ocean_lib/web3_internal/test/test_currency.py

```diff
@@ -44,16 +54,38 @@ def test_from_wei():
         with localcontext(ETHEREUM_DECIMAL_CONTEXT):
             from_wei(MAX_WEI + 1)
 
-    USDT_DECIMALS = 6
-    assert from_wei(0, USDT_DECIMALS) == Decimal(
-        "0"
-    ), "Zero wei of USDT should equal zero ether of USDT"
-    assert from_wei(123456789_123456789, USDT_DECIMALS) == Decimal(
-        "0.123456"
-    ), "Conversion from wei to ether using decimals failed"
-    assert from_wei(1123456789_123456789, USDT_DECIMALS) == Decimal(
-        "1.123456"
-    ), "Conversion from wei to ether using decimals failed"
+
+@pytest.mark.unit
+def test_format_units():
+    """Test the format_units function"""
+    assert format_units(0, USDT_DECIMALS) == Decimal("0")
+    assert format_units(123456, USDT_DECIMALS) == Decimal("0.123456")
+    assert format_units(1_123456, USDT_DECIMALS) == Decimal("1.123456")
+    assert format_units(5278_020000, USDT_DECIMALS) == Decimal("5278.02")
+    assert format_units(MIN_WEI, USDT_DECIMALS) == MIN_USDT
+    assert format_units(MAX_WEI, USDT_DECIMALS) == MAX_USDT
+
+    with pytest.raises(ValueError):
```

Comment:
Can we have `pytest.raises` tests in separate tests?

Reply:
Since there's only one error case, I think It's fine to keep them in the same test.